src/python_pybind11/test/DiffDriveOdometry_TEST.py

```diff
@@ -38,7 +39,7 @@ def test_diff_drive_odometry(self):
 
         # Setup the wheel parameters, and initialize
         odom.set_wheel_params(wheelSeparation, wheelRadius, wheelRadius)
-        startTime = datetime.datetime.now()
+        startTime = datetime.timedelta(time.monotonic())
         odom.init(datetime.timedelta())
```

Comment:
```suggestion
        startTime = datetime.timedelta(time.monotonic())
        odom.init(startTime)
```

then update
`odom.update(..., ..., time1)`